pubsub/gossipsub/README.md

```diff
@@ -298,35 +305,29 @@ for each topic in mesh:
      emit PRUNE(topic) control message to peer
 ```
 
-The `fanout` map is maintained by keeping track of last published time
-for each topic:
+The `fanout` map is maintained by keeping track of the length of time since a 
+topic was last published:
 ```
 for each topic in fanout:
- if last published time > TTL
-   remove topic from fanout
- else if |fanout[topic]| < D
-   select D - |fanout[topic]| peers from peers.gossipsub[topic] - fanout[topic]
-   add the peers to fanout[topic]
+  if time_since_last_published > ttl
```

Comment:
please don't underscorize this.

Reply:
I will remove them if you wish, however I did so because it's a variable.